mypy.ini

```diff
@@ -0,0 +1,12 @@
+[mypy]
+ignore_missing_imports = True
+disallow_incomplete_defs = True
+disallow_untyped_defs = True
+disallow_untyped_calls = True
+show_error_codes = True
+
+[mypy-tests.*]
+ignore_errors = True
+
+[mypy-evergreen.cli.*]
+ignore_errors = True
```

Comment:
Why? Since no one will consume the APIs here programmatically?

Reply:
We haven't really supported the cli code for this. It is mostly just used to manually test things and isn't really the purpose of this repository. As such, it is a lot of work to bring it up to standards. So, for now I think we should just ignore it and if someone wants to go clean it up at some point they can.